src/tests/performance/ping_pong.rs

```diff
@@ -55,53 +57,48 @@ async fn p001_t1_PING_PONG_throughput() {
-    // │      15 │        150 │          0 │         49 │              8 │          0 │          0 │          0 │          0 │         49 │          14.84 │      12.08 │        27.64 │
+    // │      50 │       1000 │          0 │        369 │              8 │          0 │          0 │          0 │          0 │         47 │         100.00 │      11.20 │      4463.01 │
     // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
-    // │      20 │        150 │          0 │         41 │              5 │          0 │          0 │          0 │          0 │         41 │          16.13 │      13.04 │        37.12 │
-    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
-    // │      30 │        150 │          0 │         23 │              2 │          0 │          0 │          0 │          0 │         11 │          13.24 │      15.40 │        38.70 │
-    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
-    // │      50 │        150 │          0 │         36 │              2 │          0 │          0 │          0 │          0 │          1 │           9.59 │      18.67 │        38.52 │
+    // │     100 │       1000 │          0 │       3130 │             48 │          0 │          0 │          0 │          1 │         44 │          71.01 │     137.67 │       515.81 │
     // └─────────┴────────────┴────────────┴────────────┴────────────────┴────────────┴────────────┴────────────┴────────────┴────────────┴────────────────┴────────────┴──────────────┘
     //
     // *NOTE* run with `cargo test --release tests::performance::ping_pong -- --nocapture`
+    // Before running test generate dummy devices with different ips using toos/ips.py
 
-    let synth_counts = vec![1, 10, 15, 20, 30, 50];
+    let synth_counts = vec![1, 10, 15, 20, 30, 50, 100];
 
     let mut table = RequestsTable::default();
 
     let target = TempDir::new().expect("Unable to create TempDir");
     let mut node = Node::builder()
+        .max_peers(MAX_PEERS)
         .start(target.path(), NodeType::Stateless)
         .await
         .unwrap();
     let node_addr = node.addr();
 
+    // This is "the hack" but is needed to perform next tests if IPS table is not empty. The
+    // standard TIME_WAIT is 60s before we can use the same addr and port again.
+    // So we're taking already used IPs and each thread in each iteration will use different IP.
+    // If the table is empty or too small, the thread itself will notice it and will use the
+    // local IP.
+    // It can be removed once pea2pea will offer REUSE_ADDR options.
+    let mut used = 0;
```

Comment:
maybe we could improve naming here
```suggestion
    let mut ip_idx = 0;  or // used_ip_cnt
```

Reply:
Done.